client/src/Pages/v1/StatusPage/Status/Components/MonitorsList/index.jsx

```
@@ -19,13 +22,20 @@ const MonitorsList = ({
 	const theme = useTheme();
 	const { determineState } = useMonitorUtils();
+	const [dateRange, setDateRange] = useState("recent");
 
 	const { showURL } = useSelector((state) => state.ui);
 
 	return (
 		<>
 			{monitors?.map((monitor) => {
 				const status = determineState(monitor);
+				const [monitorData, monitorStats, monitorIsLoading, monitorNetworkError] =
+					useFetchUptimeMonitorById({
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)